app/views/shared/_further_info_trigger.html.erb

```diff
@@ -0,0 +1,5 @@
+<a href="" class="further-info__trigger" data-further-info-trigger>
+  <svg class="svg-icon svg-icon-tooltip" xmlns="http://www.w3.org/2000/svg" style="">
```

Comment:
It has a non js fallback to hidding the link, but displaying the 'tooltip' content. We've not altered the existing behaviour of the tooltips, but good point about handling a js error.
![screen shot 2015-10-07 at 16 03 22](https://cloud.githubusercontent.com/assets/32398/10341508/f5b8d7f0-6d0c-11e5-8ecd-2216b494b869.png)